requirements.txt

```diff
@@ -1,6 +1,7 @@
 inspect_ai>=0.3.75
 streamlit>=1.42.0
 plotly>=6.0.0
+pytest>=8.3.5
```

Comment:
@eleweek this is installed from the `pyproject.toml` with `pip install .[dev]`. We don't need it here as it's not a dependency of the app itself.